rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -226,8 +255,19 @@ private void handleFindNode(FindNodePeerMessage message) {
     }
 
     private void handleNeighborsMessage(InetSocketAddress neighborsResponseAddress, NeighborsPeerMessage message) {
+        logger.debug("co.rsk.net.discovery.PeerExplorer.handleNeighborsMessage - Handling neighbors message:\n" +
+                        "neighborsResponseAddress hostName: {},\n" +
+                        "neighborsResponseAddress port: {},\n" +
+                        "nodeId: {},\n" +
+                        "messageId: {}\n" +
+                        "nodesCount: {}\n" +
+                        "nodes: {}"
+                , neighborsResponseAddress.getHostName(), neighborsResponseAddress.getPort(), message.getNodeId(), message.getMessageId(), message.countNodes(), message.getNodes());
+
         Node connectedNode = this.establishedConnections.get(message.getNodeId());
 
+        logger.debug("co.rsk.net.discovery.PeerExplorer.handleNeighborsMessage - Get connectedNode result: {}", connectedNode);
+
```

Comment:
curious if it'd be possible to combine these two logs into one? And in similar cases below when multiple occurrences of 'logger.debug(..)` appear in one method? Simply just to not overwhelm our logs?

Reply:
Done, but I left some logs like this for some `sendX()` to indicate when a message gets created, like [here](https://github.com/rsksmart/rskj/pull/2157/files#diff-3c1580dcca7d09fbf9c2ef1c09a183beed0868926163dac666303d3d0661a54eR313).